polkadot/runtime/parachains/src/coretime/migration.rs

```diff
@@ -142,8 +142,8 @@ mod v_coretime {
 			let new_core_count = assigner_coretime::Pallet::<T>::session_core_count();
 			ensure!(new_core_count == prev_core_count, "Total number of cores need to not change.");
 			ensure!(
-				dmp_queue_size == prev_dmp_queue_size + 1,
-				"There should have been enqueued one DMP message."
+				dmp_queue_size == prev_dmp_queue_size + 4,
+				"There should have been four enqueued DMP messages."
 			);
 
```

Comment:
Will the migration execute again or be prevented by `!Self::already_migrated()`?  
In general it is a bit dangerous to amend an existing migration without bumping any version.

Reply:
The migration will not execute again on Rococo, tested via `try-runtime`. The messages will be submitted manually on Rococo.